File: android/src/main/kotlin/com/sdk/bluesnap_sdk/models/CheckoutCardProps.kt
package com.sdk.bluesnap_sdk.models

data class CheckoutCardProps(var cardNumber: String,
                               var expirationDate: String,
                               var cvv: String,
                               var name: String,
                               var billingZip: String,
                               var isStoreCard: Boolean = false,
                               var email: String?)

File: ios/Classes/CheckoutCardProps.swift
//
//  CheckoutCardProps.swift
//  BluesnapSdkFlutter
//
//  Created by MacOS on 8/18/23.
//

import Foundation

class CheckoutCardProps {
    var cardNumber: String
    var expirationDate: String
    var cvv: String
    var name: String
    var billingZip: String
    var isStoreCard: Bool
    var email: String?

    init () {
        self.cardNumber = ""
        self.expirationDate = ""
        self.cvv = ""
        self.name = ""
        self.billingZip = ""
        self.isStoreCard = false
        self.email = nil
    }

}
